### PR TITLE
Rules version shouldn't be hardcoded

### PR DESCRIPTION
**Pull request by tremes**
# Description

Adding a version attribute to the `Rules` type so that the version is correctly propagated. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

## Testing steps

Just check the version in the `rules.json` vs what the running service is returning

## Checklist
* [ ] updated documentation wherever necessary
* [X] added or modified tests if necessary

### CODE REVIEW

**Comment by tremes** [comment]
@dpensi please check

**Review by dpensi — APPROVED** [review]

**Review by juandspy — APPROVED** [review]

File: tests/rest/rules_endpoint.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func readGatheringRules(f *frisby.Frisby) Payload {
 // checkResponse checks if the payload returned by server has proper format
 func checkResponse(f *frisby.Frisby, response Payload) {
 	if response.Version != "1.0" {
```
Comment (tremes) [inline tests/rest/rules_endpoint.go:54]: Does this condition make sense in the tests?

Comment (tisnik) [reply tests/rest/rules_endpoint.go:54]: needs to be replaced by using the actual tag

Comment (dpensi) [reply tests/rest/rules_endpoint.go:54]: there's no actual tag, it is comparing against this sample file here [tests/conditions/rules.json](https://github.com/RedHatInsights/insights-operator-gathering-conditions-service/pull/123/files#diff-cc8f42e6dec07f07b21db1de2ad4397f90cbafc911d34c900419b473ec9a6a02) where I added an arbitrary version

**Pull request merged**